pandas/_libs/groupby.pyx

```diff
@@ -51,7 +51,14 @@ from pandas._libs.missing cimport checknull
 cdef int64_t NPY_NAT = get_nat()
 _int64_max = np.iinfo(np.int64).max
 
-cdef float64_t NaN = <float64_t>np.NaN
+cdef:
+    float32_t MINfloat32 = np.NINF
+    float64_t MINfloat64 = np.NINF
+
+    float32_t MAXfloat32 = np.inf
+    float64_t MAXfloat64 = np.inf
```

Comment:
nit: Would prefer the INF/NEGINF naming like in the other file since max/min has a different meaning than inf

Reply:
EDIT: whoops that's the same naming as the other file